src/Orbits.jl

```diff
@@ -1,194 +1,385 @@
 using LinearAlgebra
-using Symbolics
-using IfElse
+
+###########################################################
+### 
+### Standard orbital parameter calculations and conversions
+### 
+###########################################################
 
 """
-    kepler_a_from_P(P,m1,m2)
+    kepler_a_from_P(;m_1, m_2, P)
 
 Obtain semimajor axis from period using Kepler's third law
 
-#Arguments:
+# Arguments:
+- m_1: mass of first companion [Msun]
+- m_2: mass of 2nd companion   [Msun]
 - P:  orbital period           [d]
-- m1: mass of first companion [Msun]
-- m2: mass of 2nd companion   [Msun]
 
-#Output:
+# Output:
 - a: semi-major axis of the orbit [cm]
 """
-function kepler_a_from_P(P,m1,m2)
-    return cbrt((P*day_in_sec)^2*cgrav*(m1+m2)*m_sun/(4.0*π^2))
+function kepler_a_from_P(;m_1, m_2, P)
+    return cbrt((P*day_in_sec)^2*cgrav*(m_1+m_2)*m_sun/(4.0*π^2))
 end
 
-
 """
-    kepler_P_from_a(a,m1,m2)
+    kepler_P_from_a(;m_1, m_2, a)
 
 Obtain period from semimajor axis using Kepler's third law
 
-#Arguments:
-- a:  semi-major axis of the orbit [cm]
-- m1: mass of first companion     [Msun]
-- m2: mass of 2nd companion       [Msun]
+# Arguments:
+- m_1: mass of first companion      [Msun]
+- m_2: mass of 2nd companion        [Msun]
+- a:  semi-major axis of the orbit  [cm]
 
-#Output:
+# Output:
 - P: the orbital period            [d]
 """
-function kepler_P_from_a(a,m1,m2)
-    return sqrt(4.0*π^2*a^3/(cgrav*(m1+m2)*m_sun))/(day_in_sec)
+function kepler_P_from_a(;m_1, m_2, a)
+    return sqrt(4.0*π^2*a^3/(cgrav*(m_1+m_2)*m_sun))/(day_in_sec)
+end
+
+
+"""
+    average_orbital_velocity(;m_1, m_2, a)
+
+Calculate the average orbital velocity, or equivalently
+the fixed orbital velocity for a circular orbit.
+
+# Arguments:
+- m_1: mass of first companion        [Msun]
```

Comment:
Maybe CGS for everything? A bit weird to mix solar units and centimeters

Reply:
We also use days for period and km/s for velocities. We could switch to CGS for everything, but it will just require converting / deconverting in a bunch of places, and may add computational overhead. I would lean towards using whatever units are the most intuitive for a given use case.

Reply:
I guess then the choice for each input is just what makes more sense in our minds to discuss those properties. If so perhaps one could do the inverse and use Rsun for the orbital separation?